core/lib/libtomcrypt/ecc.c

```diff
@@ -12,40 +12,7 @@
 
 #include "acipher_helpers.h"
 
-TEE_Result crypto_acipher_alloc_ecc_keypair(struct ecc_keypair *s,
-					    size_t key_size_bits __unused)
-{
-	memset(s, 0, sizeof(*s));
-	if (!bn_alloc_max(&s->d))
-		goto err;
-	if (!bn_alloc_max(&s->x))
-		goto err;
-	if (!bn_alloc_max(&s->y))
-		goto err;
-	return TEE_SUCCESS;
-err:
-	crypto_bignum_free(s->d);
-	crypto_bignum_free(s->x);
-	crypto_bignum_free(s->y);
-	return TEE_ERROR_OUT_OF_MEMORY;
-}
-
-TEE_Result crypto_acipher_alloc_ecc_public_key(struct ecc_public_key *s,
-					       size_t key_size_bits __unused)
-{
-	memset(s, 0, sizeof(*s));
-	if (!bn_alloc_max(&s->x))
-		goto err;
-	if (!bn_alloc_max(&s->y))
-		goto err;
-	return TEE_SUCCESS;
-err:
-	crypto_bignum_free(s->x);
-	crypto_bignum_free(s->y);
-	return TEE_ERROR_OUT_OF_MEMORY;
-}
-
-void crypto_acipher_free_ecc_public_key(struct ecc_public_key *s)
+static void ltc_ecc_free_public_key(struct ecc_public_key *s)
```

Comment:
Better reserve the `ltc_` prefix for the "official" LTC API and use something else here.
I propose to  add a leading underscore. We have this already for DT parsing functions: libfdt defines `fdt_` and we have our own "extensions" in `core/include/kernel/dt.h` called `_fdt_get_status()`, `_fdt_reg_size()`, `_fdt_reg_base_address()`. But I am open to other suggestions.

Reply:
No problem to use prefix `_ltc_` instead of `ltc_`, I think it better to keep the `ltc` key work in the function name when debugging with trace or when dumping disassembly file. 
It help to see quickly the function executed. When more than one function have the same name it's more painful to debug.